### PR TITLE
Negotiations: everyone + audited-payload

### PR DESCRIPTION
**Pull request by MrDarthShoe**
Resolves https://github.com/golemfactory/yagna/issues/2396

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -52,7 +52,7 @@ ya-utils-networking = "0.2"
 ya-fd-metrics = { path = "utils/fd-metrics" }
 ya-version = "0.2"
 ya-vpn = "0.2"
-ya-client = "0.7.1"
```
Comment (etam) [inline Cargo.toml:55]: downgrade?

Comment (MrDarthShoe) [reply Cargo.toml:55]: `ya-client` is actually in 0.7.0 right now https://github.com/golemfactory/ya-client/tags

Comment (MrDarthShoe) [reply Cargo.toml:55]: And actually it should be 0.7 :)

**Review by wkargul — APPROVED** [review]
LGTM

**Pull request merged**